### DISCUSSION TITLE
Ideas and suggestions

### DISCUSSION THREAD

**Discussion by ItsPi3141** [ItsPi3141/discord-fake-avatar-decorations#4] [Ideas]
Suggest any feature ideas you have here.

**Comment by potato7YT** [comment]
A advanced mode that you can change the size of the effect and position
And the size of the image
And a custom effects tab

And a banner effects
Like the profile effects but for banners
Example:
![EvoBot-7Xy8gw2Fn](https://github.com/ItsPi3141/discord-fake-avatar-decorations/assets/85459992/e7afdbb9-e7c0-4a20-8aaa-208503ae7b60)

**Reply by harshitkamboj** [reply]
nice idea.

**Reply by ademCoin** [reply]
Very nice idea

**Reply by Dekkkkkkkyyy** [reply]
Please do more

**Comment by potato7YT** [comment]
Make a discord server 😁

**Reply by 71126ggsgshsjo01syup** [reply]
i can if get support from u all to grow the server

**Comment by grandedoubleshot** [comment]
can we contribute our avatar decoration to your site?

**Reply by 71126ggsgshsjo01syup** [reply]
yup

**Comment by harshitkamboj** [comment]
Add search option.
Where we can search decoration or full collection.

**Reply by ItsPi3141** [reply]
I added the search feature. You can search for decorations by name, description, or category.

**Comment by HammouBe** [comment]
what about this:
https://cdn.discordapp.com/avatar-decoration-presets/a_e132d6014f2075d9fc2a8ece507ef5cf.png

**Reply by ItsPi3141** [reply]
that's already on the website. check the quests category.

**Comment by vladyxd** [comment]
> Suggest any feature ideas you have here.

if pfp is selected, hide default ones, so we can only see the effects

**Comment by SyntaxSpecter-0** [comment]
Being able to scale the effect so they don't get cut off when using on discord

**Reply by MurtazaNaqviCoder** [reply]
We can scale it from discord when uploading the profile

**Reply by SyntaxSpecter-0** [reply]
you can zoom in but not out on discord, some effects get clipped out

**Comment by ABDKING531** [comment]
THIS IS VERY NICE IDEA

**Comment by Star-dos-scripts** [comment]
Bom demais! 🇧🇷

**Comment by 0xdya** [comment]
you are the best 🍃✨

**Comment by ghanashyambudhathoki** [comment]
Great

**Comment by ghanashyambudhathoki** [comment]
Great

**Comment by MurtazaNaqviCoder** [comment]
Create a bot from which we can add these decorations to emojis from our server using commands directly.

**Comment by ademCoin** [comment]
We are all agree we need before have nitro boost and after one of choice

**Reply by ademCoin** [reply]
choose on of choice *

**Reply by ItsPi3141** [reply]
I don't quite understand what you mean. Could you explain more about this?

**Reply by ademCoin** [reply]
i mean i need have nitro boost before

**Reply by ItsPi3141** [reply]
You can get a still frame from the GIF 
https://discord-decorations.vercel.app/gif-extractor

**Comment by shoroneko4625** [comment]
A

**Comment by TuningStudio** [comment]
Use an animated banner, so the top corners don't get cut off. For example, Nibbles you can only see his mouth with an animated banner you can see his ears and it gifs of a real on-top effect.

https://ibb.co/84rYs498

**Reply by ItsPi3141** [reply]
That is a good idea. However, banners are positioned and scaled different depending on where it's viewed (i.e. mobile, desktop pop out, desktop model). So the decoration on the banner would not always be in the desired position.

**Comment by Dekkkkkkkyyy** [comment]
Would it be possible for you to make free avatar decorations?

**Reply by ItsPi3141** [reply]
You mean non-animated avatars? You can use the gif frame extractor. 
https://discord-decorations.vercel.app/gif-extractor

**Comment by Dekkkkkkkyyy** [comment]
Would it be possible to make free profile effects?

**Reply by ItsPi3141** [reply]
It won't show on your whole profile. It would just be on your banner, which is a tiny part of your profile.

**Comment by LcaylaCoder** [comment]
j

**Comment by xllcs** [comment]
hey there! awesome work :)
I would like to ask you guys, where did you find the hashes for some of decorations? for example, `owlbear_cub_snowy` is not present in the `urls.txt`. would you be kind enough to help with that? thanks!

**Comment by shadankhan221505-ui** [comment]
Brother profile effect was needed please give

**Comment by xehaxok109-creator** [comment]
https://discord-decorations.vercel.app/

**Comment by ayaanarmy29-commits** [comment]
steps

**Comment by multiart9054** [comment]
Hello! I'm new here!
